src/pages/Staking/StakesList/IncreaseStake/IncreaseStake.component.tsx

```diff
@@ -1,41 +1,61 @@
 import { utils } from 'ethers';
 
+import { useForm } from 'react-hook-form';
 import { Button } from '../../../../components/Button/Button.component';
 import { TextInput } from '../../../../components/TextInput/TextInput.component';
 import { DialogForm } from '../../../../components/DialogForm/DialogForm.component';
 import { CurrencyInput } from '../../../../components/CurrencyInput/CurrencyInput.component';
-import { InputWithButtonPillGroup } from '../../../../components/InputPillGroup/InputWithButtonPillGroup.component';
 
-import { IncreaseStakeComponentProps } from './IncreaseStake.types';
+import {
+  IncreaseStakeComponentProps,
+  IncreaseStakeFormValues,
+} from './IncreaseStake.types';
+import { ControlledInputWithButtonPillGroup } from '../../../../components/ControlledInputWithButtonPillGroup/ControlledInputWithButtonPillGroup.component';
+import { IncreaseStakeFields } from './IncreaseStake.fields';
 
 export const IncreaseStakeComponent = ({
   open,
   txFee,
   onClose,
   votingPower,
   currentStakeAmount,
-}: IncreaseStakeComponentProps) => (
-  <DialogForm
-    open={open}
-    txFee={txFee}
-    onClose={onClose}
-    title="Add To Stake"
-    leftButton={<Button>Stake</Button>}
-  >
-    <CurrencyInput
-      disabled
-      symbol="FISH"
-      value={currentStakeAmount}
-      title="Amount Currently Staked"
-    />
+}: IncreaseStakeComponentProps) => {
+  // TODO: change dateSelector and slectDate to controlled inputs by react-hook-form
+  const { control, setValue } = useForm<IncreaseStakeFormValues>({
+    defaultValues: {
+      [IncreaseStakeFields.IncreaseStakeAmount]: '',
+    },
+  });
+  return (
+    <DialogForm
+      open={open}
+      txFee={txFee}
+      onClose={onClose}
+      title="Add To Stake"
+      leftButton={<Button>Stake</Button>}
+      rightButton={
```

Comment:
remove the `rightButton` prop. It's optional and has a default that is identical to this